tools/run_tests/xds_k8s_test_driver/tests/authz_test.py

```diff
@@ -229,9 +229,10 @@ def test_plaintext_allow(self) -> None:
             self.configure_and_assert(test_client, 'never-match-port',
                                       grpc.StatusCode.PERMISSION_DENIED)
 
-        with self.subTest('07_principal_present'):
-            self.configure_and_assert(test_client, 'principal-present',
-                                      grpc.StatusCode.PERMISSION_DENIED)
+        # b/202058316
```

Comment:
Still, I would suggest add a slightly more info here.
```
# TODO(b/202058316): enable after the regression is fixed
```

Reply:
Except this has always been broken, it is just more broken now than it was broken before... I've added a comment about the rough current state of the bug. It keeps morphing.